api-proxy/handlers/*.options.mjs

```diff
@@ -0,0 +1,19 @@
+const matchReq = req => req.method === 'OPTIONS'
```

Comment:
This is the only handler that is hand-edited. All of the OPTIONS method responses were basically identical, so I consolidated them into one file.